swerve100/src/main/java/org/team100/lib/motor/turning/FalconTurningMotor.java

```diff
@@ -22,23 +23,35 @@ public static class Config {
     public FalconTurningMotor(String name, int canId) {
         m_motor = new WPI_TalonFX(canId);
         m_motor.configFactoryDefault();
+        m_motor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
         m_motor.setNeutralMode(NeutralMode.Brake);
         m_motor.setInverted(InvertType.InvertMotorOutput);
         m_motor.configStatorCurrentLimit(
                 new StatorCurrentLimitConfiguration(true, m_config.kCurrentLimit, m_config.kCurrentLimit, 0));
         m_motor.configSupplyCurrentLimit(
                 new SupplyCurrentLimitConfiguration(true, m_config.kCurrentLimit, m_config.kCurrentLimit, 0));
         SmartDashboard.putData(String.format("Falcon Turning Motor %s", name), this);
+        m_motor.configNominalOutputForward(0);
+        m_motor.configNominalOutputReverse(0);
+        m_motor.configPeakOutputForward(1);
+        m_motor.configPeakOutputReverse(-1);
+        m_motor.config_kF(0, 0.05);
+        m_motor.config_kP(0, 0.05);
+        m_motor.config_kI(0, 0);
+        m_motor.config_kD(0, 0);
     }
 
     @Override
     public double get() {
         return m_motor.get();
     }
 
-    //THIS DOES NOT ACTUALLY SET PID This is just here for the CAN turning motor
-    public void setPID(ControlMode control, double output) {
-        this.set(output);
+    public void setPID(ControlMode control, double outputRadiansPerSec) {
```

Comment:
this should be a separate method for velocity control (see my comment in CanTurningMotor)